kratos/utilities/discont_utils.h

```diff
@@ -1287,46 +1233,41 @@ class DiscontinuousShapeFunctionsUtilities
             inv_area = 1.0 / area;
         }
 
-
         N[0] = CalculateVol(x1, y1, x2, y2, xc, yc) * inv_area;
         N[1] = CalculateVol(x2, y2, x0, y0, xc, yc) * inv_area;
         N[2] = CalculateVol(x0, y0, x1, y1, xc, yc) * inv_area;
-        //KRATOS_WATCH(N);
-        /*
-        if (N[0] >= 0.0 && N[1] >= 0.0 && N[2] >= 0.0 && N[0] <= 1.0 && N[1] <= 1.0 && N[2] <= 1.0) //if the xc yc is inside the triangle return true
-            return true;
 
-        return false;
-        */
     }
 
     static inline double CalculateVol(const double x0, const double y0,
                                       const double x1, const double y1,
-                                      const double x2, const double y2
-                                     )
+                                      const double x2, const double y2)
     {
         return 0.5 * ((x1 - x0)*(y2 - y0)- (y1 - y0)*(x2 - x0));
     }
 
     template <int TDim>
-    static void AddToEdgeAreas(array_1d<double, (TDim-1)*3 >& edge_areas, 
-                        const array_1d<double, TDim+1 >& exact_distance,
-                        const array_1d<double, TDim+1 >& indices,
-                        const double sub_volume)
+    static void AddToEdgeAreas(array_1d<double, (TDim-1)*3 >& edge_areas,
+                               const array_1d<double, TDim+1 >& exact_distance,
+                               const array_1d<double, TDim+1 >& indices,
+                               const double sub_volume)
     {
         //check if the element has 3 "nodes" on the cut surface and if the remaining one is positive
         //to do so, remember that edge nodes are marked with an id greater than 3
         unsigned int ncut=0, pos=0, positive_pos=0;
         for(unsigned int i=0; i<TDim+1; i++)
         {
-            if(indices[i] > TDim) ncut++;
+            if(indices[i] > TDim)
+            {
+                ncut++;
+            }   
             else if(exact_distance[indices[i]] > 0)
             {
                 positive_pos = indices[i];
                 pos++;
             }
         }
-               
+
         if(ncut == TDim && pos==1) //cut face with a positive node!!
         {
             double edge_area = sub_volume*3.0/fabs(exact_distance[positive_pos]);
```

Comment:
https://stackoverflow.com/questions/33738509/whats-the-difference-between-abs-vs-fabs-in-c